app/views/twoeye_confirm_error.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
```

Comment:
We also need a test class for this view.

Reply:
done, included in next commit